### ISSUE TITLE
Accessibility: Instructions are not associated

### ISSUE THREAD

**Issue by vikrampm1** [woothemes/woocommerce#51832]
**Prerequisites (mark completed items with an [x]):**
- [x] I have checked that my issue type is not listed here https://github.com/woocommerce/woocommerce/issues/new/choose
- [x] My issue is not a security issue, support request, bug report, enhancement or feature request (Please use the link above if it is).

### Issue Description:

Additional information/ instruction is required to interact with the control, but is not associated to assistive technology users.

![T007_SR_InstructionNotAssociated](https://github.com/user-attachments/assets/886368aa-7f1b-46b9-8cd1-06dd8e3a4c67)

### Component or Page: 

Account details My Account Page

### Name affected element:

Text below the Display name field, 'This will be how your name will be displayed in the account section and in reviews'

### Relevant code snippets:

`<p class=""woocommerce-form-row woocommerce-form-row--wide form-row form-row-wide"">
		<label for=""account_display_name"">Display name&nbsp;<span class=""required"">*</span></label>
		<input type=""text"" class=""woocommerce-Input woocommerce-Input--text input-text"" name=""account_display_name"" id=""account_display_name"" value=""eva-maria adam""> <span><em>This will be how your name will be displayed in the account section and in reviews</em></span>
	</p>`

### Recommended fix:

Make sure the instruction/additional information, 'This will be how your name will be displayed in the account section and in reviews' is associated with the display name field. Add an aria-describedby attribute to the input field that references the helper text.
